lib/server.js

```diff
@@ -37,6 +37,7 @@ function Server (torrent, opts) {
     })
 
     // Only call `server.close` if user has not called it already
+    if (!cb) cb = function () {}
```

Comment:
It's best to set all defaults at the top of the function instead of in the body.